docs/tools/cli/service/schema-registry-acl.rst

```diff
@@ -23,13 +27,17 @@ Adds a Karapace schema registry ACL entry. The detailed explanation of parameter
   * - ``service_name``
     - The name of the service
   * - ``--permission``
-    - The permission type: possible values are ``schema_registry_read`` or ``schema_registry_write``
+    - The permission type: 
+  
+      - ``schema_registry_read``
+      -  ``schema_registry_write``
   * - ``--resource``
-    - The resource to grant access to, can be in the form of ``Config:`` or ``Subject:<subject>``, accepts ``*`` and ``?`` as wildcard characters. More information can be found in the :ref:`dedicated page <karapace_schema_registry_acls>`.
+    - The resource format can be ``Config:`` or ``Subject:<subject>``. For more information, see `ACLs definition <karapace_schema_registry_acls>`.
   * - ``--username``
-    - The username pattern: accepts ``*`` and ``?`` as wildcard characters
+    - The name of a service user
 
-**Example:** Add an ACL for users with username starting with ``userAB`` to write (``schema_registry_write``) subjects having name starting with ``s123`` in the Aiven for Apache Kafka® service ``kafka-doc``.
+**Example**
+The following example shows you how to add an ACL entry to grant a user (``user_1``) read options (``schem_registry_read``) to subject ``s1``. Replace the placeholders ``PROJECT_NAME`` and ``APACHE_KAFKA_SERVICE_NAME`` with the name of the project and the Aiven for Apache Kafka® service.
```

Comment:
The text doesn't match the command. We have `user_1` vs `userAB*` (is this a wildcard?) and `schema_registry_write` vs (mispelled) `schem_registry_read`

Reply:
Thanks, corrected this.